nengo/neurons.py

```diff
@@ -298,8 +300,9 @@ class LIF(LIFRate):
 
     min_voltage = NumberParam('min_voltage', high=0)
 
-    def __init__(self, tau_rc=0.02, tau_ref=0.002, min_voltage=0):
-        super(LIF, self).__init__(tau_rc=tau_rc, tau_ref=tau_ref)
+    def __init__(self, tau_rc=0.02, tau_ref=0.002, amplitude=1, min_voltage=0):
```

Comment:
Yeah, people shouldn't rely on the position of keyword arguments. But there's also no need to have ``amplitude`` in front of ``min_voltage``, so I'll switch them.